src/SweetSFSymbolsView.ts

```diff
@@ -0,0 +1,3 @@
+// Sweet-SFSymbols is not supported on any platform other than iOS
+
+export default () => null
```

Comment:
Everything looks good, just one small suggestion. Without this, typescript complains that there is a mismatch in types.

```suggestion
// Sweet-SFSymbols is not supported on any platform other than iOS
import { SweetSFSymbolsViewProps } from "./SweetSFSymbols.types";

export default (props: SweetSFSymbolsViewProps) => null;

```